vignettes/S4VectorOverview.Rmd

```diff
@@ -0,0 +1,369 @@
+---
+title: "**An Overview of the S4Vectors package**"
+author: 
+- name: "Patrick Aboyoun"
+- name: "Michael Lawrence"
+- name: "Hervé Pagès"
+- name: "Beryl Kanali"
+  affiliation: "Converted 'S4VectorOverview' vignette from Sweave to RMarkdown."
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: S4Vectors
+vignette: >
+  %\VignetteIndexEntry{An Overview of the S4Vectors package}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}

```

Comment:
Add to this vignette section: 
%\VignetteKeywords{Vector,Hits,Rle,List,DataFrame}
%\VignettePackage{S4Vectors}